src/excmds.ts

```diff
@@ -4709,6 +4902,18 @@ export function jumble() {
         body.currentNode.textContent = jumble_helper(t)
     }
 }
+
+/**
+ * Run a command in a tab with an internal firefox identifier specified by destination or in background if destination is "background".
+ */
+//#content
+export function acceptExCmd(destination: number | "background", ex_string: string[]) {
```

Comment:
```suggestion
export function acceptExCmd(destination: number | "background", ...ex_string: string[]) {
```

without this we can't run it interactively from the Tridactyl command line